core/frontend/src/PerModelCategoryVisibility.ts

```diff
@@ -103,27 +112,77 @@ class PerModelCategoryVisibilityOverrides extends SortedArray<PerModelCategoryVi
       return PerModelCategoryVisibility.Override.None;
   }
 
-  public setOverride(modelIds: Id64Arg, categoryIds: Id64Arg, override: PerModelCategoryVisibility.Override): void {
+  /**
+   *
+   * @param perModelCategoryVisibility array of model category visibility overrides
+   * @param viewState Optional param viewState. If no viewState is provided, then the viewState associated with the viewport (used to construct this class) is used. This param is WIP, not sure if necessary.
```

Comment:
Update this comment when you get a response.

Reply:
I spoke with someone from the OCP team, and they said it would help their use case.